website_google_tag_manager/models/res_config_settings.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2016 ABF OSIELL <http://osiell.com>
+# Copyright 2018 Tecnativa - Cristina Martin R.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    has_google_tag_manager = fields.Boolean(
+        'Google Tag Manager',
+        related='website_id.has_google_tag_manager')
+    google_tag_manager_key = fields.Char(
+        'ID Google Tag Manager',
```

Comment:
How Google names this? It's good to have the same nomenclature for easy recognition of the parameters. Starting with, this is not the same label as the related one. If you don't put any, this takes the one from related field, so I think it's more convenient to remove it and be sure on the other part.

Reply:
Yes, I'll change to "container ID"